node/coinstacks/common/api/src/websocket.ts

```diff
@@ -48,35 +22,35 @@ export interface TxsTopicData {
   addresses: Array<string>
 }
 
+export interface MessageResponse {
+  address: string
+  data: string
+  subscriptionId: string
+}
+
 export interface Methods {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  subscribe: (subscriptionId: string, data?: any) => Promise<void>
+  subscribe: (subscriptionId: string, data?: any) => void
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   unsubscribe: (subscriptionId: string, data?: any) => void
 }
 
 export class ConnectionHandler {
   public readonly clientId: string
 
-  private readonly rabbit: Connection
   private readonly websocket: WebSocket
+  private readonly registry: Registry
   private readonly routes: Record<Topics, Methods>
-  private readonly unchainedExchange: Exchange
-  private readonly txExchange: Exchange
+  private readonly pingInterval = 10000
```

Comment:
is 10 seconds often enough?

Reply:
Been the standard ping interval I have been using. We could make it shorter if we want.